docs/run-transition-locally.md

```diff
@@ -0,0 +1,484 @@
+bin/vaultclient generate-account-access-key --name account1 --port 8600
+```
+
+```sh
+aws configure --profile account1
+```
+
+### Create a lifecycle transition bucket
+
+```sh
+aws s3api create-bucket --bucket local-bucket --create-bucket-configuration LocationConstraint=s3c-location --endpoint-url http://127.0.0.1:8000 --profile account1
+```
+
+Create `transition.json`
+
+```json
+{
+    "Rules": [
+        {
+            "Status": "Enabled",
+            "Prefix": "",
+            "Transitions": [{
+                "Days": 1,
+                "StorageClass": "aws-location"
+            }],
+            "ID": "456"
+        }
+    ]
+}
+```
+
+```sh
+aws s3api put-bucket-lifecycle-configuration --endpoint-url http://127.0.0.1:8000 --profile account1 --bucket local-bucket --lifecycle-configuration file://transition.json
+```
+
+### Create the management account
+
+This management account will hold all the services accounts.
+Those services user will assume the customer accounts's role to gain permission access.
+
+```sh
+bin/vaultclient create-account --name management --email dev@null --port 8600 --accountid 000000000000
+```
+
+```sh
+bin/vaultclient generate-account-access-key --name management --port 8600
+```
+
+```sh
+aws configure --profile management
+```
+
+### Create the lifecycle service user
+
+NOTE: We could also use `bin/ensureServiceUser` to create the service user.
+
+When creating the service user, we can make sure that the lifecycle service user's arn matches the
+`accountSeeds.role.trustPolicy.Statement.Principal.AWS`
+(i.e. `arn:aws:iam::000000000000:user/lifecycle`) defined in the `config.json` in the Vault section.
+Indeed, the lifecycle service user is the only user trusted to assume the customer account's role.
+
+```sh
+aws iam create-user --user-name lifecycle --endpoint-url http://127.0.0.1:8600 --profile management
+```
+
+```sh
+aws iam create-access-key --user-name lifecycle --endpoint-url http://127.0.0.1:8600 --profile management
+```
+
+```sh
+aws configure --profile lifecycle
+```
+
+### Allow the lifecycle service user to assume roles
+
+Create `assume.json` policy:
+
+```json
+{
+   "Version": "2012-10-17",
+   "Statement": {
+     "Effect": "Allow",
+     "Action": "sts:AssumeRole",
+     "Resource": "*"
+   }
+ }
+
+```
+
+```sh
+aws iam create-policy --policy-name assume --policy-document file://assume.json --endpoint-url http://127.0.0.1:8600 --profile management
+```
+
+```sh
+aws iam attach-user-policy --policy-arn arn:aws:iam::000000000000:policy/assume --user-name lifecycle --endpoint-url http://127.0.0.1:8600 --profile management
+```
+
+## Backbeat
+
+### Add a lifecycle conductor service user to retrieve the buckets’ account id
+
+This user will assume its local role with the conductor policy.
+
+AWS_PROFILE=management bin/ensureServiceUser apply backbeat-lifecycle-conductor -p extensions/lifecycle/conductor/policy.json --iam-endpoint http://127.0.0.1:8600 --sts-endpoint http://127.0.0.1:8800
```

Comment:
Can you wrap this in a code block like the other commands? I would suggest it but suggesting code blocks messes up a little.